### DISCUSSION TITLE
Strange gene expression map

### DISCUSSION THREAD

**Discussion by Ping-lin14** [GreenleafLab/ArchR#794] [Questions / Documentation]
Hello,

I performed the analysis based on ArchR: 10x Multiome PBMC, but after reading the visual gene expression, I got a strange map.
Does this mean that in my data, all cells have the same gene expression level?
And I noticed that when I execute the `plotEmbedding` function, I receive the message `No imputeWeights found, returning NULL` Does it matter?
![1622534743158](https://user-images.githubusercontent.com/69965537/120412741-a8829c80-c389-11eb-9595-176d45260de4.jpg)

**Comment by rcorces** [comment]
> Does this mean that in my data, all cells have the same gene expression level?

Most likely yes. But this should be easy for you to confirm.

> I receive the message No imputeWeights found, returning NULL Does it matter?

Only if you want to use imputation
